src/components/navigation/z-user-dropdown/styles.css

```diff
@@ -1,141 +1,269 @@
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.32px;
+}
+
+.useremail.inverse {
+  color: var(--text-grey-800);
 }
 
 /* Tablet breakpoint */
 @media only screen and (min-width: 768px) {
   :host {
     height: calc(var(--space-unit) * 4);
+    background-color: var(--bg-grey-900);
+  }
+
+  :host(.inverse) {
+    background-color: var(--bg-white);
   }
 
   :host > div {
     position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+
+  :host > div.open {
+    top: auto;
+    transform: none;
   }
 
   :host > div > div {
     padding: 0;
   }
 
   :host > div > div.open {
-    display: flex;
-    flex-direction: column;
-    flex-wrap: nowrap;
-    justify-content: flex-start;
-    align-items: flex-end;
-    align-content: stretch;
-    width: min-content;
     position: absolute;
     right: calc(var(--space-unit) * -1);
     box-shadow: 0px 2px calc(var(--space-unit) * .5) 0px rgba(0, 0, 0, 0.5);
-    background: var(--bg-white);
+    background-color: var(--bg-white);
     border-radius: calc(var(--space-unit) * .5);
-    padding: calc(var(--space-unit) * .5) var(--space-unit);
+    padding: var(--space-unit) var(--space-unit) 0 var(--space-unit);
     transform: translate(0, -4px);
   }
 
+  :host > div > div.inverse.open {
+    background-color: var(--bg-grey-800);
+  }
+
   button {
     display: flex;
     flex-direction: row;
     flex-wrap: nowrap;
     justify-content: flex-end;
     align-items: end;
     align-content: center;
-    max-width: 200px;
   }
 
-  button > span {
-    padding: 0 var(--space-unit);
-  }
   button.open {
     background-color: var(--bg-white);
     border-top: var(--border-size-small) solid var(--bg-white);
     border-top-left-radius: calc(var(--space-unit) * .5);
     border-top-right-radius: calc(var(--space-unit) * .5);
     max-width: initial;
+    margin-left: var(--space-unit);
+    padding-bottom: calc(var(--space-unit) * 2);
+  }
+
+  button.open.inverse {
+    border-top-color: var(--bg-grey-800);
+    background-color: var(--bg-grey-800);
+  }
+
+  button:not(.open) z-icon {
+    fill: var(--text-white);
   }
 
-  button > .userfullname {
-    display: initial;
-    white-space: nowrap;
-    overflow: hidden;
-    max-width: 250px;
-    text-overflow: ellipsis;
+  button:not(.open) z-icon.inverse {
+    fill: var(--text-grey-800);
   }
 
-  button.open > z-icon {
-    fill: var(--gray900);
+  button.open z-icon {
+    fill: var(--text-grey-800);
+  }
+  
+  button.open z-icon.inverse {
+    fill: var(--text-white);
+  }
+
+  .userfullname {
+    font-family: var(--dashboard-font);
+    font-weight: var(--font-sb);
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.16px;
+  }
+
+  .userfullname.inverse {
+    color: var(--text-grey-800);
+  }
+
+  button.open div.userfullname {
+    color: var(--text-grey-800);
+    max-width: none;
+    text-overflow: clip;
```

Comment:
In caso di nomi molto lunghi il nome deve essere tagliato senza ellipsis?

Reply:
Non mi pare che tagli, sembra corretto il comportamento